src/main/application/TcfApiV2.js

```diff
@@ -42,8 +49,9 @@ class TcfApiV2 {
   }
 
   getVendorList(callback, vendorListVersion) {
+    const translationLanguage = 'es' // TODO: extract
```

Comment:
Do we have to fix in the window object the language to spanish? i understand that for our UI we will fix it to spanish, but it will be fixed in the UI components, not in the BorosTcf facade, but I don't see the need to fix it in the exposed to window object.